client/src/components/FancyUL/FancyUL.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import { FancyUL } from "./FancyUL.tsx";
+
+export default {
+  title: "FancyUL",
+  component: FancyUL,
+};
+
+const Template = (args) => <FancyUL {...args} />;
+
+const children = <div>Children</div>;
```

Comment:
`FancyUL` requires an array of children. Guess we should be writing storybooks in typescript to catch this stuff, haha.

Reply:
Done!